Packs/TheHiveProject/Classifiers/classifier-Hive_Classifier.json

```
@@ -0,0 +1,20 @@
+{
+ "defaultIncidentType": "Hive",
+ "feed": false,
+ "id": "30778892-2dca-4b17-884f-3b67babe3170",
```

id should be same as name. change that for all content entities.
```suggestion
 "id": "Hive_Classifier",
```

---

Done

---

You changed only to this file, change this in **all** content entities in the pack.